projects/noloan2/app/src/main/java/hasadna/noloan2/LawsuitPdfActivity.java

```diff
@@ -32,179 +39,243 @@
-    lawsuit = lawsuit.replace("&lt;companyAddress&gt;", companyAddress);
-    lawsuit = lawsuit.replace("&lt;companyPhone&gt;", companyPhone);
-    lawsuit = lawsuit.replace("&lt;companyFax&gt;", companyFax);
-
-    // Second spam company
-    lawsuit = lawsuit.replace("&lt;company2Name&gt;", company2Name);
-    lawsuit = lawsuit.replace("&lt;company2Id&gt;", company2Id);
-    lawsuit = lawsuit.replace("&lt;company2Address&gt;", company2Address);
-    lawsuit = lawsuit.replace("&lt;company2Phone&gt;", company2Phone);
-    lawsuit = lawsuit.replace("&lt;company2Fax&gt;", company2Fax);
-
-    return lawsuit;
-  }
-
-  private String getPdfFilename() {
-    return DATE_TIME_FORMATTER.format(new Date()) + ".pdf";
-  }
+    lawsuit =
+        lawsuit.replace(
+            "<userFullName>",
+            ((EditText) findViewById(R.id.userPrivateName)).getText().toString()
+                + " "
+                + ((EditText) findViewById(R.id.userLastName)).getText().toString());
+    lawsuit =
+        lawsuit.replace("<userId>", ((EditText) findViewById(R.id.userID)).getText().toString());
+    lawsuit =
+        lawsuit.replace(
+            "<userAddress>", ((EditText) findViewById(R.id.userAddress)).getText().toString());
+    lawsuit =
+        lawsuit.replace(
+            "<userPhone>", ((EditText) findViewById(R.id.userPhone)).getText().toString());
+    lawsuit =
+        lawsuit.replace("<userFax>", ((EditText) findViewById(R.id.userFax)).getText().toString());
 
-  private void initDirStructure() {
-    // TODO: Paths.get() require API 26. Find/Create function that suits lower API versions.
-    lawsuitMainPath =
-        Paths.get(
-                Environment.getExternalStorageDirectory().getPath(),
-                getString(R.string.app_name),
-                LAWSUIT_MAIN_DIR_NAME)
-            .toString();
-    makeDir(lawsuitMainPath);
+    // Spam company
+    lawsuit =
+        lawsuit.replace(
+            "<companyName>", ((EditText) findViewById(R.id.companyName)).getText().toString());
+    lawsuit =
+        lawsuit.replace(
+            "<companyId>", ((EditText) findViewById(R.id.companyId)).getText().toString());
+    lawsuit =
+        lawsuit.replace(
+            "<companyAddress>",
+            ((EditText) findViewById(R.id.companyAddress)).getText().toString());
+    lawsuit =
+        lawsuit.replace(
+            "<companyPhone>", ((EditText) findViewById(R.id.companyPhone)).getText().toString());
+    lawsuit =
+        lawsuit.replace(
+            "<companyFax>", ((EditText) findViewById(R.id.companyFax)).getText().toString());
 
-    // Output
-    lawsuitOutputPath = Paths.get(lawsuitMainPath, LAWSUIT_OUTPUT_DIR_NAME).toString();
-    makeDir(lawsuitOutputPath);
+    return lawsuit;
   }
 
-  private void makeDir(String path) {
-    File directory = new File(path);
-    if (directory.exists() && directory.isDirectory()) directory.mkdirs();
+  // Pops a dialog asking user if to share pdf
```

Comment:
Maybe change to "asking the user to share the pdf", I think it sound better.

Reply:
Wrote it like that at the beginning, but it felt a little "pushy", lol
Ok :)

Done.